### PR TITLE
add new env myriad-api

### PR DESCRIPTION
**Pull request by apronny**
# add new env myriad-api

- [N/A

# Description

add new env myriad-api

### CODE REVIEW

**Review by abdulhakim2902 — APPROVED** [review]

File: charts/myriad-api/templates/deployment.yaml [hunk 1]
```diff
@@ -129,6 +129,28 @@ spec:
             - name: MONGO_URL
               value: {{ .Values.config.mongo.url | quote }}
             {{- end }}
+            {{- if and .Values.config.redis.connector .Values.config.redis.host .Values.config.redis.port .Values.config.redis.password }}
+            - name: REDIS_CONNECTOR
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "myriad-api.redisSecretName" . }}
+                  key: connector
```
Comment (irmannmal) [inline charts/myriad-api/templates/deployment.yaml]: No need to put as secret

File: charts/myriad-api/templates/deployment.yaml [hunk 2]
```diff
@@ -129,6 +129,28 @@ spec:
...
+            - name: REDIS_HOST
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "myriad-api.redisSecretName" . }}
+                  key: host
```
Comment (irmannmal) [inline charts/myriad-api/templates/deployment.yaml]: No need to put as secret

File: charts/myriad-api/templates/deployment.yaml [hunk 3]
```diff
@@ -129,6 +129,28 @@ spec:
...
+            - name: REDIS_PORT
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "myriad-api.redisSecretName" . }}
+                  key: port
```
Comment (irmannmal) [inline charts/myriad-api/templates/deployment.yaml]: No need to put as secret

**Review by irmannmal — CHANGES_REQUESTED** [review]

File: charts/myriad-api/templates/deployment.yaml (hunk 1 above)
Comment (apronny) [reply charts/myriad-api/templates/deployment.yaml]: done

File: charts/myriad-api/templates/deployment.yaml (hunk 3 above)
Comment (apronny) [reply charts/myriad-api/templates/deployment.yaml]: done

**Review by irmannmal — APPROVED** [review]

**Pull request merged**